scoop-basics/src/main/java/com/example/scoop/basics/ui/transitions/dialogtransitions/screen/Dialog.java

```diff
@@ -9,10 +12,31 @@
 import com.lyft.scoop.ExitTransition;
 import com.lyft.scoop.Screen;
 import com.lyft.scoop.dagger.DaggerModule;
+import java.io.ByteArrayOutputStream;
 
 @Controller(DialogController.class)
 @DaggerModule(DialogModule.class)
 @EnterTransition(SlideUpTransition.class)
 @ExitTransition(SlideDownTransition.class)
 public class Dialog extends Screen {
+    private byte[] backgroundBitmap;
```

Comment:
@kathyma We should move background logic out of Screen and find better place for it.